### PR TITLE
Use `safehttpx.get()` instead of `async_get_with_secure_transport()` 

### PR DESCRIPTION
**Pull request by abidlabs**

### CODE REVIEW

File: test/test_processing_utils.py
Comment (abidlabs) [inline test/test_processing_utils.py:411]: These 10 tests are now in the `safehttpx` package

Comment (abidlabs) [reply test/test_processing_utils.py:411]: https://github.com/gradio-app/safehttpx/blob/main/test/test.py

**Review by freddyaboulton — APPROVED** [review]
Nice @abidlabs ! Left some nits in the safehttpx repo that do not block merge

**Review by aliabid94 — APPROVED** [review]
LGTM! agree w freddy that safehttp is a better package name (or ssrfsafehttp if safehttp is taken)

**Comment by abidlabs** [comment]
We should be good to merge once CI passes, thanks @freddyaboulton and @aliabid94!

**Pull request merged**
